scripts/scrapper/zendesksupport/metadata/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"log"
+
+	"github.com/amp-labs/connectors/tools/fileconv/api3"
+	"github.com/amp-labs/connectors/tools/scrapper"
+	"github.com/amp-labs/connectors/zendesksupport/metadata"
+	"github.com/amp-labs/connectors/zendesksupport/openapi"
+)
+
+func main() {
+	explorer, err := openapi.FileManager.GetExplorer()
+	must(err)
+
+	aliases := api3.NewAliases(map[string]string{
+		"search":               "results",
+		"ticket_audits":        "audits",
+		"satisfaction_reasons": "reasons",
+		// problems -> additionalProperties
```

Comment:
Some object names are stored under different alternative naming within openapi schema.
For Zendesk there are 3 exceptions.

Problems has no static fields, it is a free form map.